app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the users controller here.
```

Comment:
If you are not going to style a page individually, better remove the file and add it later when you have content to add